app/domain/logs.rb

```diff
@@ -555,6 +555,51 @@ module AuthnJwt
         code: "CONJ00109D"
       )
 
+      FetchingIdentityPath = ::Util::TrackableLogMessageClass.new(
```

Comment:
@shulifink Can you go over this messages?

CC : @hilagross

Reply:
@shulifink i put some examples of the logs in the PR comment